sdk/formrecognizer/azure-ai-formrecognizer/tests/test_copy_model.py

```diff
@@ -62,9 +62,12 @@ def test_copy_model_fail(self, client, container_sas_url, location, resource_id)
         # give an incorrect region
         target = client.get_copy_authorization(resource_region="eastus", resource_id=resource_id)
 
-        with self.assertRaises(HttpResponseError):
+        try:
             poller = client.begin_copy_model(model.model_id, target=target)
             copy = poller.result()
+        except HttpResponseError as e:
```

Comment:
I had a test similar to this and I thought it was passing, but it turns out it wasn't actually raising an `HttpResponseError`, so none of my asserts were actually being checked. Definitely recommend using `with pytest.raises`. Example [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/tests/test_extract_key_phrases.py#L99)